core/src/main/java/kafka/log/remote/quota/RLMQuotaManager.java

```diff
@@ -112,10 +110,7 @@ private Sensor sensor() {
         return sensorAccess.getOrCreate(
             quotaType.toString(),
             RLMQuotaManagerConfig.INACTIVE_SENSOR_EXPIRATION_TIME_SECONDS,
-            sensor -> {
-                sensor.add(metricName(), new SimpleRate(), getQuotaMetricConfig(quota));
-                return BoxedUnit.UNIT;
-            }
+            sensor -> sensor.add(metricName(), new SimpleRate(), getQuotaMetricConfig(quota))
```

Comment:
`SensorAccess` is used by RLM also, hence it would be better to move `SensorAccess` to server-common module.

Reply:
good idea, done.